cmd/olympus/actions/storage.go

```diff
@@ -24,19 +26,19 @@ func GetStorage() (storage.BackendConnector, error) {
 			return nil, fmt.Errorf("missing disk storage root (%s)", err)
 		}
 		s := fs.NewStorage(rootLocation, afero.NewOsFs())
```

Comment:
With the recent PR #199 this call now returns an error so will need to be updated